### DISCUSSION TITLE
How i format my disk in Cosmos?

### DISCUSSION THREAD

**Discussion by FelekDevYT** [CosmosOS/Cosmos#3026] [Q&A] [answered]
How i format my disk in Cosmos?

**Comment by blackneeed** [comment, marked as answer]
for (int i = 0; i < YourCosmosVFSInstance.Disks[0].Partitions.Count; i++)
{
    YourCosmosVFSInstance.Disks[0].DeletePartition(i);
}
YourCosmosVFSInstance.Disks[0].Clear();
YourCosmosVFSInstance.Disks[0].CreatePartition(SIZE_OF_PARITION_IN_MB);
YourCosmosVFSInstance.Disks[0].FormatPartition(0, "FAT32", true);

Keep in mind. this deletes all partitions on first disk, clears it, creates a partition, and formats it

**Comment by Gabolate** [comment]
@Blackneeed You know what, this should be in a FAQ, cuz i've seen this more than twice

**Comment by blackneeed** [comment]
@FelekDevYT Can u close this so i dont explode? :D
